### PR TITLE
Fixes and additions to user docs

### PR DESCRIPTION
**Pull request by LappleApple**
**What changed?**
Made changes to a number of "user guide" pages.

**Why was this change made?**
To improve user experience and accuracy.

**Release notes**
Made changes to a number of "user guide" pages to improve experience.

### CODE REVIEW

File: website/docs/getting-started/ui.mdx [hunk 1]
```diff
@@ -109,4 +109,4 @@ Finally let's look at the Source in more detail - go back to the Details tab, an
 
 As with an Application detail view, you can see key information about how the resource is defined.
 
-Now we are familiar with the Dashboard, let's deploy a new application :sparkles:.
+Now that we are familiar with the WGE dashboard, let's deploy a new application :sparkles:.
```
Comment (yiannistri) [inline website/docs/getting-started/ui.mdx:112]:
```suggestion
Now that we are familiar with the dashboard, let's deploy a new application :sparkles:.
```

Although the description applies to both OSS and WGE dashboards, the screenshots we use are from OSS.

File: website/docs/intro-ee.mdx [hunk 2]
```diff
@@ -8,7 +8,7 @@ import Link from "@docusaurus/Link";
 # Weave GitOps Enterprise Edition <TierLabel tiers="Enterprise" />
 
 :::tip Ready for more GitOps?
-To purchase entitlement to Weave GitOps Enterprise Edition please contact [sales@weave.works](mailto:sales@weave.works)
+To purchase an entitlement to Weave GitOps Enterprise Edition please contact [sales@weave.works](mailto:sales@weave.works)
```
Comment (yiannistri) [inline website/docs/intro-ee.mdx:11]:
```suggestion
To purchase an entitlement to Weave GitOps Enterprise please contact [sales@weave.works](mailto:sales@weave.works)
```

File: website/docs/intro-ee.mdx [hunk 3]
```diff
@@ -22,31 +22,30 @@ If you have already purchased your entitlement, head to the [installation page](
 ## Feature breakdown
 
 In addition to the features in the OSS edition, Weave GitOps Enterprise Edition
```
Comment (yiannistri) [inline website/docs/intro-ee.mdx:24]:
```suggestion
In addition to the features in the OSS edition, Weave GitOps Enterprise
```

File: website/docs/installation/weave-gitops-enterprise/index.mdx [hunk 4]
```diff
@@ -15,7 +15,7 @@ import oauthAzureDevOpsSuccess from '/img/oauth-azure-devops-success.png';
 ## Installing Weave GitOps Enterprise<TierLabel tiers="Enterprise" />
 
 :::info
-To purchase entitlement to Weave GitOps Enterprise Edition please contact [sales@weave.works](mailto:sales@weave.works)
+To purchase an entitlement to Weave GitOps Enterprise Edition, please contact [sales@weave.works](mailto:sales@weave.works).
```
Comment (yiannistri) [inline website/docs/installation/weave-gitops-enterprise/index.mdx:18]:
```suggestion
To purchase an entitlement to Weave GitOps Enterprise, please contact [sales@weave.works](mailto:sales@weave.works).
```

File: website/docs/installation/weave-gitops-enterprise/index.mdx [hunk 5]
```diff
@@ -129,6 +138,7 @@ flux bootstrap github \
   --branch=main \
   --path=./clusters/management \
   --personal
+  --components source-controller,kustomize-controller,helm-controller,notification-controller,image-reflector-controller,image-automation-controller
```
Comment (yiannistri) [inline website/docs/installation/weave-gitops-enterprise/index.mdx:141]:
```suggestion
  --components-extra image-reflector-controller,image-automation-controller
```
Flux installs source-controller, kustomize-controller, helm-controller and notification-controller by default so I'd suggest using this flag to specify only the additional components.

File: website/docs/installation/weave-gitops.mdx [hunk 6]
```diff
@@ -79,21 +97,19 @@ Let's get into it... :sparkles:
      --branch=main \
      --path=./clusters/my-cluster \
      --personal
+     --components source-controller,kustomize-controller,helm-controller,notification-controller,image-reflector-controller,image-automation-controller
```
Comment (yiannistri) [inline website/docs/installation/weave-gitops.mdx:100]:
```suggestion
     --components-extra image-reflector-controller,image-automation-controller
```

**Review by yiannistri — COMMENTED** [review]
These documentation fixes make a lot of sense to me 👌  I have suggested some minor edits, please take a look.

File: website/docs/getting-started/ui.mdx [hunk 7]
```diff
@@ -16,8 +16,8 @@ Weave GitOps provides insights into your application deployments, and makes cont
    ```
 
 1. [Open the dashboard](http://localhost:9001/) and login using either the [emergency cluster user](../configuration/emergency-user.mdx)
-	or OIDC based on your [configuration](../configuration/securing-access-to-the-dashboard.mdx).
-	If you followed the example above, the emergency user will be configured with the username set to `admin`.
+	or the OIDC, based on your [configuration](../configuration/securing-access-to-the-dashboard.mdx).
```
[the remark this answers is not in the document]
Comment (LappleApple) [reply website/docs/getting-started/ui.mdx]: +1

File: website/docs/installation/weave-gitops-enterprise/index.mdx
Comment (yiannistri) [inline website/docs/installation/weave-gitops-enterprise/index.mdx:1]: A suggestion for a follow-up PR: we should move away from CAPD and use the [vcluster provider](https://github.com/loft-sh/cluster-api-provider-vcluster) as an example in order to simplify our docs. (vcluster does not require access to the underlying docker engine in the same way that CAPD does)

**Review by yiannistri — APPROVED** [review]
Looks great ✨

File: website/docs/installation/weave-gitops-enterprise/index.mdx
Comment (LappleApple) [reply website/docs/installation/weave-gitops-enterprise/index.mdx:1]: +1 -- I address this in my notes, as @steve-fraser had the same comment. Thanks so much for your review, @yiannistri!

**Pull request merged**